docs/Measuring_Resonances.md

```diff
@@ -1,48 +1,68 @@
 # Measuring Resonances
 
-Klipper has built-in support for ADXL345 accelerometer, which can be used to
-measure resonance frequencies of the printer for different axes, and auto-tune
-[input shapers](Resonance_Compensation.md) to compensate for resonances.
-Note that using ADXL345 requires some soldering and crimping. ADXL345 can be
-connected to a Raspberry Pi directly, or to an SPI interface of an MCU
-board (it needs to be reasonably fast).
-
-When sourcing ADXL345, be aware that there is a variety of different PCB
-board designs and different clones of them. Make sure that the board supports
-SPI mode (small number of boards appear to be hard-configured for I2C by
-pulling SDO to GND), and, if it is going to be connected to a 5V printer MCU,
-that it has a voltage regulator and a level shifter.
-
+Klipper has built-in support for the ADXL345 and MPU-9250 compatible
+accelerometers which can be used to measure resonance frequencies of the printer
+for different axes, and auto-tune [input shapers](Resonance_Compensation.md) to
+compensate for resonances. Note that using accelerometers requires some
+soldering and crimping. The ADXL345 can be connected to the SPI interface of a
+Raspberry Pi or MCU board (it needs to be reasonably fast). The MPU family can
+be connected to the I2C interface of a Raspberry Pi directly, or to an I2C
+interface of an MCU board (it needs to support 400kbit/s *fast mode*).
```

Comment:
We should add a table of supported MCUs since only rp2040 and avr chips work.

Reply:
See latest version for table. I tried to be concise but still meaningful. Hopefully it is clear. [I need someone to give me a STM32 board at some point so I can patch them for 400kbit/s I2C.]